samples/GuestConfiguration/audit-installed-application-linux/README.md

```diff
@@ -0,0 +1,42 @@
+# Audit for installed application - Linux
+
+This policy audits the results of a check for applications installed in Linux virtual machines.
+
+This policy requires another definition to also be deployed that loads requirements for performing audits inside virtual machines.  Please see the policy sample
+[Deploy extension to audit for installed applications - Linux](..\deployextension-installed-application-linx\README.md).
+
+## Azure Policy Docs
+
+See more information and a complete walk-through of using this sample on
+[docs.microsoft.com](https://docs.microsoft.com/azure/governance/policy/samples/audit-installed-application-linux).
+
+## Try with Azure portal
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/?#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FGuestConfiguration%2Faudit-installed-application-linux%2Fazurepolicy.json)
+[![Deploy to Azure Gov](https://docs.microsoft.com/azure/governance/policy/media/deploy/deployGovbutton.png)](https://portal.azure.us/?#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FGuestConfiguration%2Faudit-installed-application-linux%2Fazurepolicy.json)
+
+## Try with Azure PowerShell
+
+````powershell
+# Create the Policy Definition (Subscription scope)
+$definition = New-AzPolicyDefinition -Name 'audit-installed-application-linux' -DisplayName 'Audit that an application is installed inside Linux VMs' -description 'This policy governs the Audit that an application is installed inside Linux VMs' -Policy 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/GuestConfiguration/audit-installed-application-linux/azurepolicy.rules.json' -Parameter 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/GuestConfiguration/audit-installed-application-linux/azurepolicy.parameters.json' -Mode All
```

Comment:
the -description param doesn't make sense in this cmdlet

Reply:
same for CLI

Reply:
fixed both.  pluralized since technically this does support a comma delimited list.